src/AutofacSerilogIntegration/SerilogContainerBuilderExtensions.cs

```diff
@@ -19,11 +19,15 @@ public static class SerilogContainerBuilderExtensions
         /// <param name="autowireProperties">If true, properties on reflection-based components of type <see cref="ILogger"/> will
         /// be injected.</param>
         /// <param name="dispose"></param>
+        /// <param name="onlyKnownConsumers">
+        /// If true, only the registrations that can be verified to use <see cref="ILogger"/> will be modified to access the logger,
+        /// thus avoiding unnecessary logger calls.
```

Comment:
Maybe "unnecessary reflection"?

Reply:
It's actually not about reflection.

```cs
var log = args.Context.Resolve<ILogger>().ForContext(registration.Activator.LimitType);
```

Probably "resolution" would be most appropriate.

(BTW, this line has quite a potential for improving, because you don't have to resolve `ILogger` from container, you have all the information you need right here - and you'll skip redundant `NamedParameter` check as well)